### PR TITLE
remove Akita - organization - Feat 7284

### PR DESCRIPTION
**Pull request by phpgeekfr**
remove Akita - organization - Feat 7284

### CODE REVIEW

File: apps/crm/crm/src/app/crm/crm.module.ts [hunk 1]
```diff
@@ -114,6 +115,7 @@ export const panelRoutes: Routes = [
     GetEventPipeModule,
     ClipboardModule,
     DetailedTermsModule,
+    UploadWidgetModule,
```
Comment (phpgeekfr) [inline apps/crm/crm/src/app/crm/crm.module.ts:118]: Why this did not created any issue earlier ??

File: libs/organization/src/lib/organization/+state/organization.service.ts [hunk 2]
```diff
@@ -174,4 +176,10 @@ export class OrganizationService extends CollectionService<OrganizationState> {
 
     this.update(orgState.id, { wishlist });
   }
+
+  public isAddedToWishlist(movieId: string): Observable<boolean> {
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/+state/organization.service.ts:180]: rename `isInWishlist`. I'm not 100% sure it's the best place for it though. But that's ok

File: libs/organization/src/lib/organization/guard/pending-organization.guard.ts [hunk 3]
```diff
@@ -1,53 +1,48 @@
...
-    return this.authQuery.user$.pipe(
-      switchMap(user => {
-        if (!user) return of('/');
+  canActivate() {
+    return combineLatest([
+      this.authQuery.user$,
+      this.service.org$
+    ]).pipe(
+      map(([user, org]) => {
+        if (!user) {
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/guard/pending-organization.guard.ts:27]: You don't need the braces :
```typescript
if (!user) return this.router.createUrlTree(['/']);
if (!user.orgId) return this.router.createUrlTree(['/auth/identity']);
if (!org) return this.router.createUrlTree(['/auth/identity']);
```

File: libs/organization/src/lib/organization/guard/request-access.guard.ts [hunk 4]
```diff
@@ -1,59 +1,53 @@
...
-  sync() {
-    return this.authQuery.user$.pipe(
-      switchMap(user => {
+  canActivate() {
+    return combineLatest([
+      this.authQuery.user$,
+      this.service.org$
+    ]).pipe(
+      map(([user, org]) => {
         if (!user) {
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/guard/request-access.guard.ts:27]: same as above

**Review by GrandSchtroumpf — APPROVED** [review]

**Pull request merged**